docs/current/162770-faq.md

```diff
@@ -2,12 +2,151 @@
+
+```shell
+curl -L https://dl.dagger.io/dagger/install.sh | sh
+```
+
+The above will create `./bin/dagger`.
+
+Homebrew users can alternatively use the following commands:
+
+```shell
+brew update
+brew upgrade dagger
+```
+
+#### SDKs
+
+To update a Dagger SDK, follow the same procedure that you would follow to update any other SDK package in your chosen development environment. For example:
+
+- Go SDK
+
+  ```shell
+  go get -u dagger.io/dagger
+  ```
+
+- Node.js SDK
+
+  <Tabs>
+  <TabItem value="npm">
+
+  ```shell
+  npm install @dagger.io/dagger@latest --save-dev
+  ```
+
+  </TabItem>
+  <TabItem value="yarn">
+
+  ```shell
+  yarn upgrade --latest @dagger.io/dagger --dev
+  ```
+
+  </TabItem>
+  </Tabs>
+
+- Python SDK
+
+  <Tabs>
+  <TabItem value="PyPI">
+
+  ```shell
+  pip install -U dagger-io
+  ```
+
+  </TabItem>
+  <TabItem value="Conda">
+
+  ```shell
+  conda update dagger-io
+  ```
+
+  </TabItem>
+  </Tabs>
+
+### What compatibility is there between the Dagger Engine, Dagger SDKs and Dagger CLI versions?
+
+- The Dagger CLI is released in tandem with the Dagger Engine and thus shares a version number with it.
+- Dagger SDKs automatically provision a Dagger Engine at a compatible version.
+
+Each release notes the compatible Dagger Engine version in its release notes. If running from the CLI, or providing a CLI for the SDK to use via the `_EXPERIMENTAL_DAGGER_CLI_BIN` variable, check the release notes of the SDK, which indicate the required CLI and Engine versions.
+
+The image below shows an example of the compatibility information available in the release notes:
+
+![Release information](/img/current/faq/release-notes.png)
+
+### How do I uninstall Dagger?
+
+Follow these steps:
+
+1. Remove the Dagger CLI using the following command:
+
+  ```shell
+  sudo rm /usr/local/bin/dagger
+  ```
+
+  Homebrew users can alternatively use the following command:
+
+  ```shell
+  brew uninstall dagger
+  ```
+
+1. Remove the `dagger` sub-directory of your `$XDG_CACHE_HOME` and `$XDG_CONFIG_HOME` directories.
+
+  <Tabs>
+  <TabItem value="macOS">
+
+  ```shell
+  rm -rf ~/Library/Caches/dagger
+  rm -rf ~/Library/Application\ Support/dagger
+  ```
+
+  </TabItem>
+  <TabItem value="Linux">
+
+  ```shell
+  rm -rf ~/.cache/dagger
+  rm -rf ~/.config/dagger
+  ```
+
+  </TabItem>
+  </Tabs>
+
+  :::note
+  The paths listed above are defaults and may require adjustment for your specific environment. The third-party packages Dagger uses to determine these paths are listed below:
+    - Go (SDK and CLI): [https://github.com/adrg/xdg](https://github.com/adrg/xdg)
+    - NodeJS: [https://github.com/sindresorhus/env-paths](https://github.com/sindresorhus/env-paths)
+    - Python: [https://github.com/platformdirs/platformdirs](https://github.com/platformdirs/platformdirs)
+  :::
+
+1. Remove the Dagger container using the following commands:
```

Comment:
Best move this up one point. Removing cache should be the last step because if there's anything dagger still in your system, it may write to those folders, or you may have an open lock/sock file in there before removing the engine container.

Reply:
Modified